2020-02-21 15:48:06,049:WARNING: From /Users/ayushagarwal/Downloads/starter_code/imitationlearning/q2_linear.py:54: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.

2020-02-21 15:48:06,082:WARNING: From create_demos.py:110: The name tf.variable_scope is deprecated. Please use tf.compat.v1.variable_scope instead.

2020-02-21 15:48:06,082:WARNING: From create_demos.py:112: conv2d (from tensorflow.python.layers.convolutional) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.keras.layers.Conv2D` instead.
2020-02-21 15:48:06,085:WARNING: From /Users/ayushagarwal/miniconda3/envs/cs330env/lib/python3.6/site-packages/tensorflow/python/ops/init_ops.py:1251: calling VarianceScaling.__init__ (from tensorflow.python.ops.init_ops) with dtype is deprecated and will be removed in a future version.
Instructions for updating:
Call initializer instance with the dtype argument instead of passing it to the constructor
2020-02-21 15:48:06,172:WARNING: Entity <bound method Conv.call of <tensorflow.python.layers.convolutional.Conv2D object at 0x1a46e2c358>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv.call of <tensorflow.python.layers.convolutional.Conv2D object at 0x1a46e2c358>>: AssertionError: Bad argument number for Name: 3, expecting 4
2020-02-21 15:48:06,244:WARNING: Entity <bound method Conv.call of <tensorflow.python.layers.convolutional.Conv2D object at 0x1a46e2c358>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv.call of <tensorflow.python.layers.convolutional.Conv2D object at 0x1a46e2c358>>: AssertionError: Bad argument number for Name: 3, expecting 4
2020-02-21 15:48:06,310:WARNING: Entity <bound method Conv.call of <tensorflow.python.layers.convolutional.Conv2D object at 0x1a46e2c358>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv.call of <tensorflow.python.layers.convolutional.Conv2D object at 0x1a46e2c358>>: AssertionError: Bad argument number for Name: 3, expecting 4
2020-02-21 15:48:06,377:WARNING: Entity <bound method Conv.call of <tensorflow.python.layers.convolutional.Conv2D object at 0x1a46e2c358>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv.call of <tensorflow.python.layers.convolutional.Conv2D object at 0x1a46e2c358>>: AssertionError: Bad argument number for Name: 3, expecting 4
2020-02-21 15:48:06,379:WARNING: From create_demos.py:121: flatten (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.flatten instead.
2020-02-21 15:48:06,391:WARNING: Entity <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x1a46e2c358>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x1a46e2c358>>: AttributeError: module 'gast' has no attribute 'Num'
2020-02-21 15:48:06,406:WARNING: From create_demos.py:122: dense (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.dense instead.
2020-02-21 15:48:06,495:WARNING: Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x1a46e2c358>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x1a46e2c358>>: AssertionError: Bad argument number for Name: 3, expecting 4
2020-02-21 15:48:06,590:WARNING: Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x1a46e2c358>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x1a46e2c358>>: AssertionError: Bad argument number for Name: 3, expecting 4
2020-02-21 15:48:06,660:WARNING: Entity <bound method Conv.call of <tensorflow.python.layers.convolutional.Conv2D object at 0x10e75e438>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv.call of <tensorflow.python.layers.convolutional.Conv2D object at 0x10e75e438>>: AssertionError: Bad argument number for Name: 3, expecting 4
2020-02-21 15:48:06,732:WARNING: Entity <bound method Conv.call of <tensorflow.python.layers.convolutional.Conv2D object at 0x10e75e438>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv.call of <tensorflow.python.layers.convolutional.Conv2D object at 0x10e75e438>>: AssertionError: Bad argument number for Name: 3, expecting 4
2020-02-21 15:48:06,801:WARNING: Entity <bound method Conv.call of <tensorflow.python.layers.convolutional.Conv2D object at 0x10e75e438>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv.call of <tensorflow.python.layers.convolutional.Conv2D object at 0x10e75e438>>: AssertionError: Bad argument number for Name: 3, expecting 4
2020-02-21 15:48:06,870:WARNING: Entity <bound method Conv.call of <tensorflow.python.layers.convolutional.Conv2D object at 0x10e75e438>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv.call of <tensorflow.python.layers.convolutional.Conv2D object at 0x10e75e438>>: AssertionError: Bad argument number for Name: 3, expecting 4
2020-02-21 15:48:06,884:WARNING: Entity <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x10e75e438>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x10e75e438>>: AttributeError: module 'gast' has no attribute 'Num'
2020-02-21 15:48:06,977:WARNING: Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x10e75e438>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x10e75e438>>: AssertionError: Bad argument number for Name: 3, expecting 4
2020-02-21 15:48:07,066:WARNING: Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x10e75e438>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x10e75e438>>: AssertionError: Bad argument number for Name: 3, expecting 4
2020-02-21 15:48:07,068:WARNING: From /Users/ayushagarwal/Downloads/starter_code/imitationlearning/q2_linear.py:149: The name tf.get_collection is deprecated. Please use tf.compat.v1.get_collection instead.

2020-02-21 15:48:07,068:WARNING: From /Users/ayushagarwal/Downloads/starter_code/imitationlearning/q2_linear.py:149: The name tf.GraphKeys is deprecated. Please use tf.compat.v1.GraphKeys instead.

2020-02-21 15:48:07,069:WARNING: From /Users/ayushagarwal/Downloads/starter_code/imitationlearning/q2_linear.py:153: The name tf.assign is deprecated. Please use tf.compat.v1.assign instead.

2020-02-21 15:48:07,096:WARNING: From /Users/ayushagarwal/Downloads/starter_code/imitationlearning/q2_linear.py:238: The name tf.train.AdamOptimizer is deprecated. Please use tf.compat.v1.train.AdamOptimizer instead.

2020-02-21 15:48:07,206:WARNING: From /Users/ayushagarwal/miniconda3/envs/cs330env/lib/python3.6/site-packages/tensorflow/python/ops/clip_ops.py:157: add_dispatch_support.<locals>.wrapper (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
2020-02-21 15:48:07,444:WARNING: From /Users/ayushagarwal/Downloads/starter_code/imitationlearning/core/deep_q_learning.py:132: The name tf.summary.scalar is deprecated. Please use tf.compat.v1.summary.scalar instead.

2020-02-21 15:48:07,454:WARNING: From /Users/ayushagarwal/Downloads/starter_code/imitationlearning/core/deep_q_learning.py:147: The name tf.summary.merge_all is deprecated. Please use tf.compat.v1.summary.merge_all instead.

2020-02-21 15:48:07,455:WARNING: From /Users/ayushagarwal/Downloads/starter_code/imitationlearning/core/deep_q_learning.py:148: The name tf.summary.FileWriter is deprecated. Please use tf.compat.v1.summary.FileWriter instead.

2020-02-21 15:48:08,206:WARNING: From /Users/ayushagarwal/Downloads/starter_code/imitationlearning/core/deep_q_learning.py:113: The name tf.train.Saver is deprecated. Please use tf.compat.v1.train.Saver instead.

2020-02-21 15:48:08,346:WARNING: From create_demos.py:144: The name tf.train.import_meta_graph is deprecated. Please use tf.compat.v1.train.import_meta_graph instead.

